workflow/rocoto/gefs_tasks.py

```diff
@@ -483,3 +483,25 @@ def wavepostpnt(self):
         task = rocoto.create_task(member_metatask_dict)
 
         return task
+    
+    def arch(self):
+        deps = []
+        dependencies = []
+        dep_dict = {'type': 'task', 'name': f'stage_ic'}
```

Comment:
This task should depend on the successful completion of the post-processing tasks.
For reference, take a look at `def arch` in `gfs_tasks.py` for constructing a multi-dependency  list.

Reply:
with this part of the code the arch job doesn't pass. See any issue?
```
        if self.app_config.do_wave:
            dep_dict = {'type': 'task', 'name': 'wavepostsbs'}
            deps.append(rocoto.add_dependency(dep_dict))
            if self.app_config.do_wave_bnd:
                dep_dict = {'type': 'task', 'name': 'wavepostbndpnt'}
                deps.append(rocoto.add_dependency(dep_dict))
                dep_dict = {'type': 'task', 'name': 'wavepostbndpntbll'}
                deps.append(rocoto.add_dependency(dep_dict))
            dep_dict = {'type': 'task', 'name': 'wavepostpnt'}
            deps.append(rocoto.add_dependency(dep_dict))
```